### PR TITLE
fix(tab): use MD_RIPPLE_GLOBAL_OPTIONS in tab

### PR DESCRIPTION
**Pull request by tinayuangao**

### CODE REVIEW

File: src/lib/tabs/tab-nav-bar/tab-nav-bar.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class MdTabLink {
 })
 export class MdTabLinkRipple extends MdRipple {
   constructor(elementRef: ElementRef, ngZone: NgZone, ruler: ViewportRuler,
-              @Optional() @Inject(MD_DISABLE_RIPPLES) forceDisableRipples: boolean) {
+              @Optional() @Inject(MD_RIPPLE_GLOBAL_OPTIONS) forceDisableRipples: boolean) {
```
Comment (crisbeto) [inline src/lib/tabs/tab-nav-bar/tab-nav-bar.ts]: I was just about to submit a PR for this :D. I think the parameter here has to be renamed as well.

Comment (DevVersion) [inline src/lib/tabs/tab-nav-bar/tab-nav-bar.ts]: Can we rename that to `globalOptions` as well?

Comment (tinayuangao) [inline src/lib/tabs/tab-nav-bar/tab-nav-bar.ts]: Updated. Please take another look. Thanks

**Comment by mikeumus** [comment]
:+1: 

and how do we use this exactly?

**Pull request merged**